### ISSUE TITLE
Display cross on the canvas creating a new box

### ISSUE THREAD

**Issue by aralroca** [aralroca/label-images#40]

**Comment by AnshuTrivedi** [comment]
@aralroca can i work on this issue ?
If yes then please explain what have to do in it?

**Comment by aralroca** [comment]
@AnshuTrivedi of course! Thank you a lot for the interest 😊 The idea is that in the actions of **creating a box** or **resize a box** appears a cross on the cursor that occupies the size of the entire canvas. This will help to visualize how far the box can go.


![image](https://user-images.githubusercontent.com/13313058/94711258-2fd5c700-0348-11eb-9eec-c4048df36a20.png)
